cmd/init.go

```diff
@@ -179,6 +179,13 @@ func init() {
 		defaultDashboardVersion = dashboardVersionEnv
 	}
 
+	defaultContainerRuntime := string(utils.DOCKER)
+	viper.BindEnv("container_runtime_override", "DAPR_CONTAINER_RUNTIME")
```

Comment:
I think this is not needed. These lines https://github.com/dapr/cli/blob/master/cmd/dapr.go#L100-L102 should already have a bound string, so getting that string should be good.
```
viper.GetString("container-runtime")
```

Reply:
> @ThorstenHans can you please check if the automatic env bind works as expected and use that var string instead?

Actually, it does not work. The `init` func in `cmd/init.go` is invoked before `initConfig` from `dapr/dapr.go` is called as part of `cobra.OnInitialize`.

Reply:
You can resolve it in the `Run` function of the command as `OnInitialize` is called before `Run` is called.
For example see how `network` value is got in the same `cmd/init.go` file.

Reply:
That's clear. 

However, I was a bit worried about other configuration data loaded from env vars. So, should I relocate loading data from other env vars (`DAPR_RUNTIME_VERSION`, `DAPR_DASHBOARD_VERSION`) to `Run` too?

Reply:
I agree, they can be moved to Run as well.

Reply:
@mukundansundar I moved loading configuration values from 

1. default values
2. env vars
3. flags

into a dedicated func `getConfigurationValue` and hooked it up in `PreRun` as there was already some code that is relied to configuration data.